examples/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.js

```diff
@@ -0,0 +1,18 @@
+// TODO: Update the image collection ID with the public data catalog ID
+var dataset = ee.ImageCollection('projects/openet/ensemble/conus/gridmet/monthly/v2_0')
```

Comment:
please update to OpenET/ENSEMBLE/CONUS/GRIDMET/MONTHLY/v2_0

Reply:
I did this in #305, so it's all set.